packages/utils/src/i18n.tsx

```diff
@@ -16,9 +16,15 @@ export const i18nProvider = (namespace: Language[]) => {
     {}
   );
 
-  i18n.use(LanguageDetector).use(initReactI18next).init({
-    debug: true,
-    resources,
-    fallbackLng: 'en-US',
-  });
+  i18n
+    .use(LanguageDetector)
+    .use(initReactI18next)
+    .init({
+      debug: true,
+      resources,
+      fallbackLng: 'en-US',
+      detection: {
+        order: ['navigator'],
+      },
```

Comment:
Is there a reason to override the [LanguageDetector defaults](https://github.com/i18next/i18next-browser-languageDetector/blob/9efebe6ca0271c3797bc09b84babf1ba2d9b4dbb/src/index.js#L11-L13)? These are useful for testing, and already include navigator (after many other places it checks.)